src/OrchardCore/OrchardCore.Data/OrchardCoreBuilderExtensions.cs

```diff
@@ -66,10 +67,23 @@ public static OrchardCoreBuilder AddDataAccess(this OrchardCoreBuilder builder)
                             var shellOptions = sp.GetService<IOptions<ShellOptions>>();
                             var option = shellOptions.Value;
                             var databaseFolder = Path.Combine(option.ShellsApplicationDataPath, option.ShellsContainerName, shellSettings.Name);
-                            var databaseFile = Path.Combine(databaseFolder, "yessql.db");
                             Directory.CreateDirectory(databaseFolder);
+
+                            var connectionString = shellSettings["ConnectionString"];
+                            if (connectionString != null && connectionString.StartsWith("Data Source="))
+                            {
+                                var connectionStringBuilder = new SqliteConnectionStringBuilder(connectionString);
+                                connectionStringBuilder.DataSource = Path.Combine(databaseFolder, connectionStringBuilder.DataSource);
```

Comment:
The relative path should be resolved based on `databaseFolder`.

Reply:
Check that if an absolute path is set it will work correctly (`/home/user1/db`)

Reply:
I will check ...

Reply:
Seems it should work according to https://docs.microsoft.com/en-us/dotnet/standard/data/sqlite/connection-strings#data-source